src/packages/files/files.ts

```diff
@@ -173,4 +180,71 @@ export default (app: Express) => {
     const files = await fastGlob(`${fileStorePath}/**/*`);
     res.json(files);
   });
+
+  /**
+   * @swagger
+   * /dictionary:
+   *   post:
+   *     consumes:
+   *       - multipart/form-data
+   *     description: If a file of the same name and location already exists, the new file overwrites the old
+   *     parameters:
+   *       - description: Session token used for authorization
+   *         in: header
+   *         name: x-auth-sso-token
+   *         required: true
+   *         schema:
+   *           type: string
+   *     produces:
+   *       - application/json
+   *     requestBody:
+   *       content:
+   *         multipart/form-data:
+   *          schema:
+   *            type: object
+   *            properties:
+   *              file:
+   *                format: binary
+   *                type: string
+   *     responses:
+   *       200:
+   *         description: UploadFileResponse
+   *     summary: Upload a command dictionary
+   *     tags:
+   *       - Command
+   */
+  app.post(
+    '/dictionary',
+    filesLimiter,
+    auth,
+    upload.any(),
+    async (req, res) => {
+      try {
+        const [file] = req.files as Express.Multer.File[];
+        const data = new FormData();
+        data.set('file', await fileFromPath(file.path));
+
+        //forward off to the commanding server
+        await fetch(
+          `${COMMANDING_SERVER_URL}:${COMMANDING_SERVER_PORT}/dictionary`,
```

Comment:
I'm a little confused at this API since it diverges from how the other file uploading on this server works. 
The purpose of the current file endpoints is to (1) add files to the filesystem, and (2) update the DB file tables where necessary. Why are we adding a file here if we are just forwarding it to another service?

Reply:
It was the requirement by @patkenneally. We want to only expose endpoints for our services at 2 locations:

Aerie-gateway
Hasura

The user should only go through these two locations to interact with all our aerie services.